src/index.js

```diff
@@ -1,9 +1,11 @@
 import assign from 'object-assign';
 import pascalCase from 'pascal-case';
+import React from 'react';
```

Comment:
I had the false belief that peerDependencies was deprecated in NPM3, but it's not, it's just that its behaviour changed.

This indeed seems like a good use-case for `peerDependencies`.